k8s/crds/kops.k8s.io_clusters.yaml

```diff
@@ -2407,6 +2407,31 @@ spec:
                   description: AmazonVPCNetworkingSpec declares that we want Amazon
                     VPC CNI networking
                   properties:
+                    env:
+                      description: Env is a list of environment variables to set in
+                        the container.
+                      items:
+                        description: EnvVar represents an environment variable present
+                          in a Container.
+                        properties:
+                          name:
+                            description: Name of the environment variable. Must be
+                              a C_IDENTIFIER.
```

Comment:
Would it be helpful to either give an example of what a `C_IDENTIFIER` looks like/link? `Must be a C_IDENTIFIER (see https://reference_material for more information). I personally wasn't able to use the hint in the description because I don't remember any C, and was only somewhat successful in figuring out what this format should be with my googlemachine. 

I also would not be opposed to truncating this description to the first sentence. :)

Reply:
The descriptions in these CRD fields actually come from the new API type we added a while back in #8455 specifically for this PR as well as #8402 

I copied those descriptions from upstream: https://github.com/kubernetes/api/blob/master/core/v1/types.go#L1827-L1846

I'm fine with truncating the descriptions but it will need to be done on the api type itself so that `make crds` will update these new fields to match.

https://github.com/kubernetes/kops/blob/f400f74a3e498201efc77833b297184376710d61/pkg/apis/kops/cluster.go#L646-L661